data/json/items/gun/22.json

```diff
@@ -86,15 +86,7 @@
     "clip_size": 1,
     "reload": 200,
     "barrel_volume": "360 ml",
-    "valid_mod_locations": [
-      [ "muzzle", 1 ],
-      [ "sling", 1 ],
-      [ "stock mount", 1 ],
-      [ "grip mount", 1 ],
-      [ "rail mount", 1 ],
-      [ "sights mount", 1 ],
-      [ "underbarrel mount", 1 ]
-    ],
+    "valid_mod_locations": [ [ "sling", 1 ], [ "sights mount", 1 ] ],
```

Comment:
If it’s s pipe where will you attach the sling that isn’t interfering with the pipe barrel? Like I could see s single point sling attached to the handle but I don’t think we have limited sling mounts like that currently?

Reply:
I had it under the assumption that the sling would be tied around the barrel

Reply:
I think tying both ends around the barrel would be sufficient, if letting it slide everywhere.

Reply:
> I think tying both ends around the barrel would be sufficient, if letting it slide everywhere.

```
{
    "type": "recipe",
    "activity_level": "MODERATE_EXERCISE",
    "result": "rifle_22",
    "category": "CC_*",
    "subcategory": "CSC_*_NESTED",
    "skill_used": "fabrication",
    "skills_required": [ [ "traps", 1 ], [ "gun", 1 ] ],
    "difficulty": 3,
    "time": "30 m",
    "reversible": true,
    "autolearn": true,
    "proficiencies": [ { "proficiency": "prof_gunsmithing_basic" }, { "proficiency": "prof_gunsmithing_improv" } ],
    "qualities": [ { "id": "SAW_M_FINE", "level": 1 }, { "id": "SCREW_FINE", "level": 1 } ],
    "components": [ [ [ "pipe", 1 ] ], [ [ "2x4", 1 ] ], [ [ "scrap", 2 ] ] ]
  },
```
Since our pipes are threaded and therefore don't have a knob at the end to catch on that 100% means the sling wouldn't work and your knot would just slide right off the end of the pipe.  It's gonna take more.  Heck this recipe is just flat missing lots of updated items needed.

Reply:
Also just remembered the pipe hitch knot but again the recipe is awful as is lol.